tests/test_discretisation/test_arbitrary_discretiser.py

```diff
@@ -122,3 +122,9 @@ def test_error_if_not_permitted_value_is_errors():
     age_dict = {"Age": [0, 10, 20, 30, np.inf]}
     with pytest.raises(ValueError):
         ArbitraryDiscretiser(binning_dict=age_dict, errors="medialuna")
+
+
+def test_error_if_not_binning_dict_dict_type():
```

Comment:
we'd use parametrize and test more than 1 value, so for example HOLA, 1, False, and we also need to assert that the error message is the expected one

Reply:
Done - added 2 additional test for integer and boolean and assert correct messages in all three cases

Reply:
this needs to be reformatted with parametrize, example here: https://github.com/feature-engine/feature_engine/blob/8cfab264e679c09fe85fc49cbc25c053b80b4fd2/tests/test_creation/test_decision_tree_features.py#L50